tests/model_registry/rbac/conftest.py

```diff
@@ -159,18 +161,116 @@ def model_registry_group_with_user(
         yield group
 
 
+@pytest.fixture(scope="package")
+def user_credentials_rbac() -> dict[str, str]:
+    random_str = generate_random_name()
+    return {
+        "username": f"test-user-{random_str}",
+        "password": f"test-password-{random_str}",
+        "idp_name": f"test-htpasswd-idp-{random_str}",
+        "secret_name": f"test-htpasswd-secret-{random_str}",
+    }
+
+
 @pytest.fixture(scope="session")
-def test_idp_user_session() -> Generator[UserTestSession, None, None]:
+def original_user() -> str:
+    current_user = run_command(command=["oc", "whoami"])[1].strip()
+    LOGGER.info(f"Original user: {current_user}")
+    return current_user
+
+
+@pytest.fixture(scope="module")
+def created_htpasswd_secret(
```

Comment:
does it make sense to have this at this scope? the data for the user_credentials has a wider scope.
I also think this forces us into having a single instance of the idp/user/etc. in the scope, is that what we actually want?

Reply:
You are correct. I see a scope mismatch. I will change the test_idp_user_session and associated fixtures to module. Currently none of these are being used anywhere outside of rbac, so we should either stick with module or package scoped fixtures. If that changes, we can always update the scopes.